.github/workflows/lint-and-test.yml

```diff
@@ -1,6 +1,12 @@
 name: lint-and-test
 
-on: ["push", "pull_request"]
+on:
+  push:
+  pull_request:
+    types:
+      - opened
+
```

Comment:
We can remove the extra spacing here to make the YAML look good.

Reply:
Done. And ... do you approve otherwise of this PR?